server/channels/app/import_functions.go

```diff
@@ -884,6 +886,21 @@ func (a *App) importBot(rctx request.CTX, data *imports.BotImportData, dryRun bo
 		} else {
 			bot.OwnerId = owner.Id
 		}
+	} else {
+		// we should have at least one system admin to assign as the owner
+		// and we are going to assign the oldest system admin for that.
+		admins, err := a.getAllSystemAdmins()
+		if err != nil {
+			return err
+		}
+		if len(admins) == 0 {
+			return model.NewAppError("importBot", "app.import.import_bot.no_system_admin.error", nil, "", http.StatusInternalServerError)
+		}
+		slices.SortFunc(admins, func(a, b *model.User) int {
+			return cmp.Compare(a.CreateAt, b.CreateAt)
+		})
```

Comment:
Can we memoize this with a `sync.Once` to happen only once?

Also, why do we need to sort?

Reply:
The admins are sorted with usernames, but I think it would probably better to get the oldest admin as it should be the pseudo owner.

BTW, I think `sync.Once` should make a lot of sense here.